prompb/remote.proto

```diff
@@ -21,6 +21,7 @@ import "gogoproto/gogo.proto";
 
 message WriteRequest {
   repeated prometheus.TimeSeries timeseries = 1 [(gogoproto.nullable) = false];
+  repeated prometheus.MetricMetadata metadata = 3 [(gogoproto.nullable) = false];
```

Comment:
Wrong link? I don't get the response

Reply:
Has Callum said, it is already used in Cortex.

The reason for using nullable (from [gogoproto](https://godoc.org/github.com/gogo/protobuf/gogoproto)) false is to avoid the protobuf definition having a pointer (and thus helping on allocations as [suggested by Tom](https://github.com/prometheus/prometheus/pull/6815#discussion_r386490139)). This will allow for nil values without any problems for both systems.

Reply:
Can we stick a reserved field in for 2 please?  With a comment as to why. https://developers.google.com/protocol-buffers/docs/proto#reserved

Reply:
Done. Does that wording make sense?

Reply:
I mean we are fixing Cortex mistake here I guess (I might be missing some context?), but since we are nice open-source citizens LGTM (: 

nullable = false FTW  as well. :+1:

Reply:
I don't get the comment though. Can we be explicit that Cortex already used that for some reason? If that's an ok reason we are agreeing here.

Reply:
Done. Let me know if anything else pops up.